### ISSUE TITLE
Support commit message configuration

### ISSUE THREAD

**Issue by AnandChowdhary** [all-contributors/all-contributors#374]
**Is your feature request related to a problem? Please describe.**
It'd be a great idea if we can specify `"commitMessage"` in `.all-contributorsrc` and the bot would make a commit with that message, overwriting the hardcoded messages based on commitConventions.

**Describe the solution you'd like**
Add a key for `commitMessage` in bot configuration.

**Comment by AnandChowdhary** [comment]
I now see that `commitTemplate` is a CLI argument, which may be used for this.

**Comment by AnandChowdhary** [comment]
Quick update: Adding `commitTemplate` to `.all-contributorsrc` does not change the commit message done by the bot. Perhaps because this option is not passed to the CLI config?

**Comment by AnandChowdhary** [comment]
Some quick [Sourcegraph](https://github.com/sourcegraph/sourcegraph) that might help with debugging:

1. In https://github.com/all-contributors/all-contributors-cli/blob/master/src/util/git.js#L65, `options.commitTemplate || commitTemplate` is used, so if `commitTemplate` is part of the passed `options`, it should work.
2. This function is called by the CLI in https://github.com/all-contributors/all-contributors-cli/blob/master/src/cli.js#L76, using the arguments provided by `yargv ` in https://github.com/all-contributors/all-contributors-cli/blob/master/src/cli.js#L176.
3. Now we just need to make sure that the bot passes `commitTemplate` to the CLI as an argument that `yargv` can pick up.

**Comment by Berkmann18** [comment]
Thanks for raising the issue.
Have you had the chance to look into fixing that in a PR?

**Comment by C0ZEN** [comment]
+1.

This would be awesome because I use Semantic Pull Request action and the bot's commit are not validated.
See for example this [PR failing](https://github.com/Sonia-corporation/il-est-midi-discord/pull/784).

**Comment by Berkmann18** [comment]
@C0ZEN Even with `commitConvention` set to the right commit convention (e.g. `angular`)?

**Comment by C0ZEN** [comment]
@Berkmann18 yes, my project is configured with the Angular commit convention. 
I think that the issue here is the absence of scope even if it is optional inside the [Commit Message Format](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-header).
So this has nothing to do with this repository - even if adding a scope to the commits would be better from my point of view and would fix my issue also -.

**Comment by C0ZEN** [comment]
@Berkmann18 I was about to open an issue inside Semantic Pull Request repository then I just remembers that I did override the default scopes... 😞 
And the `docs` scope is not allowed.
I will make the changes but I am confident about resolving the issue.
Sorry for the wrong lead.

**Comment by Berkmann18** [comment]
> even if adding a scope to the commits would be better from my point of view and would fix my issue also -.

Yeah, that's a good point.

> `docs` prefix is not allowed.

Not allowed where? It works with SR in projects with the Angular commit convention.

**Comment by C0ZEN** [comment]
@Berkmann18 I did not test it yet but I changed the configuration of my project to allow the type `docs` for the [Semantic Pull Request](https://github.com/zeke/semantic-pull-requests#configuration) action installed.
This is the one arguing about not having proper Angular conventional commits inside my PR and I think that this is no longer the case with this change.
So I think that I do not have a problem anymore and no need to change anything on this repository.

Though adding a scope to the bot commit like `docs(contributors): ...` does not seem to be a bad idea but I am not well placed to know if there is a possible drawback.

**Comment by Berkmann18** [comment]
> This is the one arguing about not having proper Angular conventional commits inside my PR and I think that this is no longer the case with this change.

Ah, I see, so more of a GH Action issue then 🤔.

> Though adding a scope to the bot commit like docs(contributors): ... does not seem to be a bad idea but I am not well placed to know if there is a possible drawback.

AFAIK, there won't be any issues caused by adding the scope especially when SR-maintained changelogs usually omit `docs` changes.

**Comment by C0ZEN** [comment]
> Ah, I see, so more of a GH Action issue then 🤔.

Exactly.

> AFAIK, there won't be any issues caused by adding the scope especially when SR-maintained changelogs usually omit `docs` changes.

Very nice then. 💯
